plugins/translog-s3/build.gradle

```diff
@@ -0,0 +1,46 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+/*
+ * Licensed to Elasticsearch under one or more contributor
```

Comment:
This license text block is not required for new files. Only the first block is required.

Reply:
took the one under `repository-s3`

Reply:
`repository-s3` was part of Elasticsearch that is why the text says `Licensed to Elasticsearch`. For any new file in OpenSearch, this is not required.